exonum/src/blockchain/config.rs

```diff
@@ -74,6 +74,28 @@ pub struct ConsensusConfig {
 impl ConsensusConfig {
     /// Default value for max_message_len.
     pub const DEFAULT_MAX_MESSAGE_LEN: u32 = 1024 * 1024; // 1 MB
+
+    /// Checks if propose timeout is less than round timeout. Warns if fails.
+    pub fn check_timeout_recommendations(&self) {
```

Comment:
Shouldn't it be private function? Are there any usage scenarios when it's useful for users?

Reply:
It is used in the `exonum-configuration` service. 🙃 

I propose to mark this  function as `doc(hidden)`. Additionally, I'm thinking of making this function (name) a little bit more generic. Probably, `validate_configuration` name would be better.